### ISSUE TITLE
osbs build-logs --follow doesn't stream new logs

### ISSUE THREAD

**Issue by pbabinca** [DBuildService/osbs#104]
I want to get logs for a running build. When I issue:
```
osbs --verbose build-logs --follow build-id-20150526-084136
```
I don't get any logs after the call nor any new logs appear as they are produced later during the wait.

I know that the logs are generated because
```
osbs --verbose build-logs build-id-20150526-084136
```
splits out build logs produced so far (and quits as expected for this command).

**Comment by TomasTomecek** [comment]
I suspect this is related to #102.

**Comment by TomasTomecek** [comment]
Should be resolved by #103.

**Comment by TomasTomecek** [comment]
Verified. Reopen if the issue persists.
